src/System.Management.Automation/engine/ExtraAdapter.cs

```diff
@@ -16,220 +15,6 @@
 namespace System.Management.Automation
 {
     /// <summary>
```

Comment:
Why is this deleted? If it's Windows PowerShell only now - it should be a different PR.